CMakeLists.txt

```diff
@@ -1,15 +1,82 @@
+# cmake version
 cmake_minimum_required(VERSION 3.5)
-project(docker)
 
-set(CMAKE_C_STANDARD 99)
-set(DOCKER_INCLUDE_DIR src PARENT_SCOPE)
+#cmake policy
+cmake_policy(SET CMP0015 NEW)
 
-# Set target name and source files
-file (GLOB DOCKER_SOURCE_FILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} src/**)
+# Project name.
+project(cdocker)
```

Comment:
A bit of a stickler -- given that the repository name and the link on the Docker website is to libdocker, can we just keep everything to docker instead of cdocker? If you feel strongly, we should probably rename repository and talk to Docker about changing their listing to keep everything consistent.

Reply:
I think docker could also work but the only hiccup I though could arise is the presence of docker command which is also ```docker```. But anyway, .so won't have any conflict with that, so I think changing ```cdocker``` to ```docker``` sounds sane.